darts/metrics/metrics.py

```diff
@@ -2873,11 +3049,11 @@ def mql(
         The (sequence of) actual series.
     pred_series
         The (sequence of) predicted series.
-    q
-        The quantile (float [0, 1]) of interest for the loss.
     intersect
         For time series that are overlapping in time without having the same time index, setting `True`
         will consider the values only over their common time interval (intersection in time).
+    q
+        The quantile (float [0, 1]) or list of quantiles of interest to compute the metric on.
```

Comment:
does not mention the option of passing a tuple.

Reply:
I didn't want to mention this in the API because it's what the wrappers gives, and maybe too complicated for the users.
I could add it in the description, but I feel like they should only use float, list[float]